test/fetchers/bitmart.spec.ts

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
```

Comment:
What do you think about writing a helper here with mockFetcherResponse(filename), mockUsdPrice(value) methods?